src/CMakeLists.txt

```diff
@@ -119,7 +121,8 @@ INSTALL(TARGETS createrepo_c DESTINATION bin/)
 INSTALL(TARGETS mergerepo_c DESTINATION bin/)
 INSTALL(TARGETS modifyrepo_c DESTINATION bin/)
 INSTALL(TARGETS sqliterepo_c DESTINATION bin/)
+ENDIF (NOT SKBUILD)
```

Comment:
I think we the binaries could be included with the Python package *in theory*, but when I tried it it couldn't find the shared library.  No need to worry about it for now.